.github/scripts/group-commit-message.awk

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/awk -f
+
+function add_to_group(key, value) {
+    groups[key] = groups[key] "- " value "\n"
+}
+
+{
+    if (NF > 0) {  # Skip empty lines
+        if (tolower($1) == "feat:") {
+            add_to_group("Features:", substr($0, index($0, $2)))
+        } else if (tolower($1) == "fix:") {
+            add_to_group("Fixes:", substr($0, index($0, $2)))
+        } else if (tolower($1) == "chore:") {
+            add_to_group("MISC:", substr($0, index($0, $2)))
```

Comment:
Is this intentional for this repo? We generally don't report `chore` tasks in the changelogs.